internal/controller/controlplaneauth/controller.go

```diff
@@ -157,9 +156,9 @@ func (c *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 		return managed.ExternalCreation{}, errors.New(errNotControlPlaneAuth)
 	}
 
-	a, err := c.accounts.Get(ctx, c.profile.ID)
```

Comment:
@muvaf with the old Implementation you can't use this endpoint - because its only working with User Token not with Personal Access Token (PTA) / API Token for that reason i changed the  implementation to get the UserId from JWT PayLoad UserId